app/models/tag.rb

```diff
@@ -1,4 +1,4 @@
 class Tag < ApplicationRecord
-  has_many :taggings
   has_many :jobs, through: :taggings
+  has_many :taggings
```

Comment:
In cases like this where you're not sure how to set up the join table associations at model level, I'd hope you reference the ActiveRecord docs for direction